packages/SwingSet/src/devices/plugin/device-plugin.js

```diff
@@ -53,6 +53,7 @@ export function buildRootDeviceNode(tools) {
    */
   async function createConnection(mod, index, epoch) {
     try {
+      // eslint-disable-next-line @jessie.js/no-nested-await
       const modNS = await endowments.import(mod);
```

Comment:
Why do you believe this one is safe? How would you update the rule to admit only safe awaits in this position?

Reply:
IIUC, that has already been updated to account for the safety of for-of-await loops, which is great!

I am wary of making it to complicated, as it would need to be to account for all the reasons at https://github.com/Agoric/agoric-sdk/pull/6526 . Btw, IIRC, the next one I would automate is terminal-control-flow, as it is easy for the programmer to understand why it is safe, and to anticipate that it would be judged safe.

Reply:
Sorry, wrong URL. Hold on for a bit and then reload.

Reply:
To be honest, I naïvely assumed that we were comfortably with this already-committed code and did not look critically at the sites. That said, I also don't think I understand the what kind of "await hazards" we are trying to avoid—some example failures at https://gist.github.com/turadg/e7d7645789218173a578508ae7640365#file-terminal-combined-control-flow would probably help.

I refactored initializeSwingset.js to avoid the need for some suppressions by replacing `await` with `then`/`catch`, but that's probably not we want. If the warnings are legitimate rather than just noise, then this PR is indeed a step backwards.

Reply:
Replacing this `await` with `then`/`catch` would change the scheduling characteristics to eliminate the hazard. It would then also not be semantics preserving, which may or may not be a problem, depending. If we're fine with the new scheduling semantics, then it probably is what we want.